daemon/api.go

```diff
@@ -817,6 +822,7 @@ func snapRevert(inst *snapInstruction, st *state.State) (string, []*state.TaskSe
 }
 
 func snapEnable(inst *snapInstruction, st *state.State) (string, []*state.TaskSet, error) {
+	// TODO: bail if revision is given (and != current?)
```

Comment:
Enable and disable make no sense. We should return an error. We disable/enable the snap, not a revision of it.